### ISSUE TITLE
Clicking on a button after submitting guess should link to that repo

### ISSUE THREAD

**Issue by AndrewNolte** [benawad/stripcode#9]

**Comment by igoro00** [comment]
Yes! I had to searched manually multiple times for a repo I wanted to look up after submitting my guess. That would be so much more convenient and I thinks it'd be a great way to credit the author of the repo.

I would add it myself but I don't know Elixir and can't be bothered learning another f...... web technology rn...

**Comment by igoro00** [comment]
and I can't even select the repo name to just paste in google. I have to manually type it like a caveman 😠
